Plugins/FrontendUserManagement/Services/RegistrationService.php

```diff
@@ -32,6 +33,8 @@ public function register(string $email, string $password) {
             $this->entityManager()->create($user);
             $userDetails = UserDetail::create(['user' => $user]);
             $userAddress = UserAddress::create(['user' => $user]);
+            $userProfile = InsertionProfile::create(['user' => $user]);
```

Comment:
@20gaLois17 Der Hotfix sollte noch mal überarbeitet werden.
Durch diese Änderung entsteht eine Abhängig zum Insertion Plugin (damit müsste immer auch das Insertion-Plugin installiert werden) und diese ist zudem noch zirkular.

Besser wäre anstelle der direkten InsertionProfile Erzeugung & Persistierung, eine Eventbasierte Herangehensweise.
An der Stelle ein Event feuern und im Insertion-Plugin in der Event-Behandlung das InsertionProfile erzeugen & persistieren.